### DISCUSSION TITLE
Add 'Copy Link' Icon for Quick URL Sharing in Docs

### DISCUSSION THREAD

**Discussion by Olyno** [supabase/supabase#29311] [Feature Requests]
# Improve documentation

## Link 

/

## Describe the problem

The sidebar navigation demands horizontal scrolling to find content, followed by copying links for sharing. Adding a 'Copy Link' icon to titles could simplify this process, improving usability.

## Describe the improvement

Enhance usability by introducing a 'Copy Link' icon next to titles, allowing for easier sharing of direct links to specific content, and reducing the need for manual scrolling and link copying from the sidebar.

## Additional context

/

**Comment by CodeAnk2829** [comment]
@Olyno could you please provide the screenshot as well?

**Comment by Olyno** [comment]
I'm adding details because I realized that it was available on other pages of the documentation. This problem appears mainly in the [sdk documentation](https://supabase.com/docs/reference/javascript/insert) @CodeAnk2829

**Comment by encima** [comment]
@Olyno this is because the SDK docs are built in their own repo and imported into Supabase docs. The underlying docs (and build process) would need to be modified in order to address this and have the URL copyable.

**Comment by Hallidayo** [comment]
Hi everyone, due to inactivity on this issue I've moved the issue over to discussions/enhancements.
